Calendar/BookingHelpers/GraphInstaller.ps1

```diff
@@ -0,0 +1,86 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# ===================================================================================================
+# Graph Modules installer Functions
+# ===================================================================================================
+function CheckGraphBookingsModuleInstalled {
+    [CmdletBinding(SupportsShouldProcess=$true)]
+    param ()
+
+    if (Get-Command -Module Microsoft.Graph.Bookings) {
+        Write-Host "Microsoft.Graph.Bookings module is already installed."
+    } else {
+        # This is slow, to the tune of ~10 seconds, but much more complete.
+        # Check if Microsoft.Graph.Bookings module is installed
+        $ModuleInstalled = Get-Module -ListAvailable | Where-Object { $_.Name -eq 'Microsoft.Graph.Bookings' }
+
+        if ($ModuleInstalled) {
+            Write-Host "Microsoft.Graph.Bookings module is already installed."
+        } else {
+            # Check if running with administrator rights
+            $IsAdministrator = ([Security.Principal.WindowsPrincipal] [Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole]::Administrator)
+
+            if (-not $IsAdministrator) {
+                Write-Host "Please run the script as an administrator to install the Microsoft.Graph.Bookings module."
+                exit
+            }
+
+            # Ask user if they want to install the module
+            if ($PSCmdlet.ShouldProcess('Microsoft.Graph.Bookings module', 'Import')) {
+                Write-Verbose "Installing Microsoft.Graph.Bookings module..."
+
+                # Install Microsoft.Graph.Bookings module
+                Install-Module -Name Microsoft.Graph.Bookings -Force -AllowClobber
+
+                Write-Host "Done. Microsoft.Graph.Bookings module is now installed. Please re-run the script."
```

Comment:
I agree that we need to likely do this. However, we should place it in Share somehow to avoid other scripts doing this same thing over and over again. 

Need to think about this.